SteamTrade/SteamBotError.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace SteamTrade
+{
+    public class SteamBotError
+    {
+        public SteamBotErrorType type { get; private set; }
```

Comment:
Public properties should use [PascalCasing](https://msdn.microsoft.com/en-us/library/ms229043%28v=vs.110%29.aspx)